internal/common/settings.go

```diff
@@ -132,3 +143,21 @@ func readFileSourceOfTrustSettings() (*FileTrustSourceSettings, error) {
 
 	return fileSettings, nil
 }
+
+func readSpireSourceOfTrustSettings() (*SpireTrustSourceSettings, error) {
+	if !viper.IsSet("trustsource.spire.domains") {
+		return nil, errors.New("trustsource.spire.domains is required but not found")
+	}
+
+	viper.SetDefault("trustsource.spire.certLocation", "/var/run/spire/certs/")
```

Comment:
Again, this seems like a really weird place to set defaults

Reply:
Agreed, but I'd rather save settings rehaul for a different PR.

Reply:
Gotcha